setup.sh

```diff
@@ -1,35 +1,50 @@
 #!/bin/bash
 set -x
 
-# Juju and Dependencies
+# This script installs juju version 2 and all the dependencies.
+
+if [ -z "$JUJU_USER" ]; then
+  JUJU_USER=ubuntu
+fi
+
+# Refresh the potentially stale apt cache.
 apt-get update -qq
-apt-get install -qy software-properties-common
-apt-add-repository -y ppa:juju/devel
+# Install software tools such as add-apt-repository.
+apt-get install -y software-properties-common
+# devel ppa is required for juju2 beta release
+apt-add-repository -u -y ppa:juju/devel
 # stable ppa is required for charm-tools
-apt-add-repository -y ppa:juju/stable
-apt-get update -qq
+apt-add-repository -u -y ppa:juju/stable
```

Comment:
i was going off the comment from `master` that said stable was required for charm-tools.  It appears that the stable ppa has a more recent version of c-t (2.1.9):

https://launchpad.net/~juju/+archive/ubuntu/stable

Versus devel's version of c-t (2.1.2):

https://launchpad.net/~juju/+archive/ubuntu/devel

Unless you know better, I think we want stable in there to pick up the most current charm-tools.